jazz_deployments/test/test.js

```diff
@@ -1,11 +1,1106 @@
+const chai = require('chai');
 const assert = require('chai').assert;
+const expect = require('chai').expect;
+const should = require('chai').should();
+const chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+const awsContext = require('aws-lambda-mock-context');
+const AWS = require("aws-sdk-mock");
+const request = require('request');
+const sinon = require('sinon');
+
 const index = require('../index');
+const logger = require("../components/logger.js");
+const configObj = require('../components/config.js');
+const crud = require('../components/crud')();
+const errorHandler = require("../components/error-handler.js")();
+
+describe('jazz_deployments', function () {
```

Comment:
you can have multiple "describes" one for each function so its easy to group those test suites to track them and update them. Having them in one describe is hard to identify what tests are testing which unit.

Reply:
Worked as per comments.